packages/flutter/lib/src/services/flutter_version.dart

```diff
@@ -33,15 +35,17 @@ abstract final class FlutterVersion {
           ? String.fromEnvironment('FLUTTER_FRAMEWORK_REVISION')
           : null;
 
-  /// The Flutter engine revision.
+  /// The Flutter engine revision, as a (short) Git commit ID.
   static const String? engineRevision =
       bool.hasEnvironment('FLUTTER_ENGINE_REVISION')
           ? String.fromEnvironment('FLUTTER_ENGINE_REVISION')
           : null;
 
-  // This is included since [Platform.version](https://api.dart.dev/stable/dart-io/Platform/version.html)
-  // is not included on web platforms.
-  /// The Dart version used to compile the app.
+  /// The value is a [semantic versioning](https://semver.org/) string representing the
+  /// version of the current Dart runtime, possibly followed by whitespace and other
```

Comment:
"version of the current Dart runtime" => "version of the Dart runtime used to compile the app"

Reply:
```suggestion
  /// version of the Dart runtime used to compile the app, possibly followed by whitespace and other
```